src/db/model/sync.rs

```diff
@@ -0,0 +1,26 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use std::ops::Range;
+
+use serde::{Deserialize, Serialize};
+
+/// A record indicating that a milestone is completed.
+#[derive(Debug, Clone, Serialize, Deserialize)]
+#[allow(missing_docs)]
```

Comment:
I'd say we only use `#[missing_docs]` for error types, because the have their meaning documented in the error message.